CommandHelper/LocalPackages/commands/chars/firestormjacksonstein.yml

```diff
@@ -0,0 +1,21 @@
+maxHearts: 30
+buffs:
+- /effect <plr> 12 9999 1 true
+- /effect <plr> 5 9999 5 true
+chatcolor: '&6'
+class: firestormjacksonstein
+name: §cFi§ere§cst§eor§cm§8(§cJ§eS§8)
+op: 'false'
+permissions:
+  superherocity:
+  - essentials.fly
+  - powers.fireballpunch
+powers:
+- /buff
+- /fly
+- /cast fireballpunch
```

Comment:
./cast is for magic users. i suggest looking at the human torch to see how to throw fireballs